src/common/chips-list-container.jsx

```diff
@@ -1,68 +1,81 @@
-import { Box, Typography } from '@material-ui/core';
-import Button from '@material-ui/core/Button';
+import {Box, Typography} from '@material-ui/core';
 import IconButton from '@material-ui/core/IconButton';
 import ClearIcon from '@material-ui/icons/Cancel';
+import {makeStyles} from "@material-ui/core/styles";
+import {useSelector} from "react-redux";
+
+
+const useStyles = makeStyles(() => ({
+    removeAllBox: {
+        display:"flex",
+        flexDirection: "end",
+        marginInlineStart: "auto",
+        alignItems:"center",
+    },
+}));
 
 
 const ChipsListContainer = ({
-    children, show, showAllItems, showAllItemsLabel, showLessItemsLabel, toggleShowAllItems, onClear, count, title,
-}) => {
-    if (show) {
+                                children, content, show, showAllItems, showAllItemsLabel, showLessItemsLabel, toggleShowAllItems, onClear, count, title,
+                            }) => {
+    const classes = useStyles();
+    const step = useSelector(state => state.payment.form.step).toString();
+    const deleteText = "Fjern alle";
         return (
             <Box
+                visibility={!show ? "hidden": "visible"}
                 display="flex"
                 justifyContent="center"
                 p={2}
                 flexDirection="column"
                 alignItems="center"
                 flexWrap="flex"
+                width="100%"
                 mt={2}
                 mb={2}
+                height={content ==="recipient" ? 220 : 160}
                 border={1}
                 borderColor="grey.100"
                 bgcolor="grey.100"
                 borderRadius="borderRadius"
             >
-                <Typography variant="h6">{title}</Typography>
-                <Box display="flex" justifyContent="center" flexWrap="wrap" m={1}>
-                    {children}
-
-                </Box>
                 <Box
                     m={1}
                     display="flex"
-                    justifyContent="center"
-                    alignItems="center"
                     flexDirection="row"
+                    width="100%"
                 >
-                    {toggleShowAllItems
-                    && (
-                        <>
-                            <Button size="small" onClick={toggleShowAllItems}>
-                                {showAllItems
-                                    ? showLessItemsLabel
-                                    : showAllItemsLabel}
-                            </Button>
-                            <Box mr={1} ml={1}>
-                            |
-                            </Box>
-                        </>
-                    )}
-                    <IconButton size="small" onClick={onClear}>
-                        <ClearIcon />
-                    </IconButton>
-                </Box>
+                    <Typography variant="h6">{title} ({count})</Typography>
+                    <div className={classes.removeAllBox}> {
```

Comment:
You should use `Box` instead and put the styling in the props
```
<Box display="flex" flexDirection="end" ...
```

Reply:
Fixed  justifyContent="space-between" on outer box